docs/content/users/quickstart.md

```diff
@@ -333,20 +333,28 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ```bash
     git clone <your-laravel-repo>
     cd <your-laravel-project>
-    ddev config --project-type=laravel --docroot=public --php-version=8.1
+    ddev config --project-type=laravel --docroot=public --php-version=8.2
     ddev start
     ddev composer install
     ddev php artisan key:generate
     ddev launch
     ```
 
+!!!tip "How to use SQLite database for Laravel?"
+    DDEV sets a default MariaDB database to better represent a production environment.
+
+    To select the [Laravel 11 defaults](https://laravel.com/docs/11.x/releases#application-defaults) for SQLite, use this command for `ddev config`:
+    ```bash
+    ddev config --project-type=laravel --docroot=public --php-version=8.2 --omit-containers=db --disable-settings-management=true
+    ```
+
 ## Magento
 
 === "Magento 2"
 
     Normal details of a Composer build for Magento 2 are on the [Magento 2 site](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/composer.html). You must have a public and private key to install from Magento’s repository. When prompted for “username” and “password” in `composer create`, it’s asking for your public key as "username" and private key as "password".
 
-    Note that you can install the Adobe/Magento composer credentials in your global `~/.ddev/homeadditions/.composer/auth.json` and never have to find them again. See [In-Container Home Directory and Shell Configuration](../extend/in-container-configuration).
+    Note that you can install the Adobe/Magento composer credentials in your global `~/.ddev/homeadditions/.composer/auth.json` and never have to find them again. See [In-Container Home Directory and Shell Configuration](extend/in-container-configuration.md).
```

Comment:
Hmm, the other link is working fine on latest docs. https://ddev.readthedocs.io/en/latest/users/quickstart/#magento

This link works too! https://ddev--5963.org.readthedocs.build/en/5963/users/quickstart/#magento

Reply:
Yeah, I just saw a complain from local mkdocs, and fixed it.